demos/table.php

```diff
@@ -26,9 +26,15 @@
 $table->addColumn('logo_url', [new \atk4\ui\TableColumn\Image()], ['caption'=>'Our Logo']);
 
 $table->onHook('getHTMLTags', function ($table, $row) {
-    if ($row->id == 1) {
+    switch ($row->id) {
```

Comment:
`$color = [1 => 'yellow', 2 => 'grey', 3 => 'brown'][$row->id] ?? ''` would be much simpler

Reply:
well, yes it would :)